aws/container-linux/kubernetes/bastion.tf

```diff
@@ -1,3 +1,16 @@
+locals {
+  user_data = {
```

Comment:
Locals are module-scoped and you're setting `user_data` multiple times. I think you should be able to do this inline.